docs/scw.md

```diff
@@ -11,27 +11,31 @@ module avail
 module load nextflow/21.10.6
 ```
```

Comment:
I think if we're asking users to do a conda install for the nf-core tools, we might just direct them to use the conda version of Nextflow as well - that way they can stay up-to-date more easily, too.